env/types.go

```diff
@@ -16,7 +16,181 @@ import (
 	"github.com/xo/usql/text"
 )
 
-// Vars is a map of variables.
+type varName struct {
+	name string
+	desc string
+}
+
+func (v varName) String() string {
+	return fmt.Sprintf("  %s\n    %s\n", v.name, v.desc)
+}
+
+var varNames = []varName{
+	{
+		"ECHO_HIDDEN",
+		"if set, display internal queries executed by backslash commands; if set to \"noexec\", just show them without execution",
+	},
+	{
+		"ON_ERROR_STOP",
+		"stop batch execution after error",
+	},
+	{
+		"PROMPT1",
+		"specifies the standard psql prompt",
```

Comment:
usql

Reply:
or, as in below, use the proper `text.*` value ...

Reply:
Thanks, fixed.